install-mac.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -e
+
+# Note: This install script assumes that node and python is already setup
+# and is meant for setting up local development rather than CI environments
+
+if [[ $# -gt 0 ]]; then
+    echo "Usage: install-mac.sh" >&2
+    exit 1
+fi
+
+brew install leveldb
+
+pip install -U -r requirements.txt
+
+# Make npm packages available
+brew install node || brew upgrade node
+
+# Grab the latest version of Firefox ESR.
+# For security reasons it is very important to keep up with patch releases
+# of the ESR, but a major version bump needs to be tested carefully.
+# Older ESRs are not supported by geckodriver.
+firefox_version="$(curl 'https://ftp.mozilla.org/pub/firefox/releases/' |
+grep '/pub/firefox/releases/52.' |
+tail -n 1 | sed -e 's/.*releases\///g' | cut -d '/' -f1)"
+
+wget "https://ftp.mozilla.org/pub/firefox/releases/${firefox_version}/mac/en-US/Firefox ${firefox_version}.dmg" -O firefox.dmg
+
+#npm install get-firefox
+#npx get-firefox -b esr -p mac -t firefox.dmg
+
+rm -rf Firefox.app
+hdiutil attach -nobrowse -mountpoint /Volumes/firefox-tmp firefox.dmg
+cp -r /Volumes/firefox-tmp/Firefox.app .
+hdiutil detach /Volumes/firefox-tmp
+rm firefox.dmg
+
+# Selenium 3.3+ requires a 'geckodriver' helper executable, which is not yet
+# packaged. `geckodriver` 0.16.0+ is not compatible with Firefox 52. See:
+# https://github.com/mozilla/geckodriver/issues/743
+# npm geckodriver 1.5.x = geckodriver 0.15.0
+npm install geckodriver@1.5.0
```

Comment:
Why not use the mac geckodriver from a Github release like we do for linux (i.e., https://github.com/mozilla/geckodriver/releases/download/v0.15.0/geckodriver-v0.15.0-macos.tar.gz)?

We'll be able to drop the `jpm` build tools with the ESR 60 upgrade and depending on the work required for #221 we can drop nodejs as a dependency entirely.

Reply:
It gives linux, mac and windows users the same installation procedure. Also, if the geckodriver releases location changes in the future, the install command will still be the same in openwpm (granted that the npm package is still maintained).

Note: I [started off](https://github.com/citp/OpenWPM/pull/219/files#diff-691148d686ec41672cee309128557845R30) by using https://www.npmjs.com/package/get-firefox to fetch the latest firefox esr as well, so using npm to install geckodriver as well was the logic choice. Later, I reverted away from get-firefox for this particular PR since get-firefox can't download older versions of firefox esr on demand (https://github.com/freaktechnik/get-firefox/issues/61). In the future, we may want to use it for downloading the latest esr version easily across linux, mac and windows.